### PR TITLE
Add new getOrElse as well as gerOrNull

### PR DESCRIPTION
**Pull request by Globegitter**
Here the new methods as we talked in #61

### CODE REVIEW

**Comment by Globegitter** [comment]
@kittinunf I am testing now everything imaginable, so not sure why the coverage is still showing up as read. It is also marking that I am not testing any line of the new `getOrElse` method but I am in fact testing both branches. Is it maybe an issue with codecov?

**Comment by kittinunf** [comment]
Just only that sir! Also, I need to investigate more what's going on with codecoverage. But that should not prevent this from being merged

File: result/src/main/kotlin/com/github/kittinunf/result/Result.kt [hunk 1]
```diff
@@ -14,11 +14,28 @@ infix fun <V : Any, E : Exception> Result<V, E>.or(fallback: V) = when (this) {
...
+}
+
+fun <V: Any, E: Exception> Result<V, E>.getOrNull(): V? {
+    return when (this) {
+        is Result.Success -> value
+        is Result.Failure -> null
+    }
+}
+
+
```
[the remark this answers is not in the document]
Comment (Globegitter) [reply result/src/main/kotlin/com/github/kittinunf/result/Result.kt]: Done :)

**Comment by Globegitter** [comment]
@kittinunf addressed the comment and also added the method to `SuspendableResult`. Let me know if there is anything else.

**Comment by kittinunf** [comment]
No sir! Let's merge this! Thanks 🤗

**Pull request merged**